packages/mui-material/src/Select/Select.js

```diff
@@ -114,7 +114,7 @@ const Select = React.forwardRef(function Select(inProps, ref) {
     ...(multiple && native && variant === 'outlined' ? { notched: true } : {}),
     ref: inputComponentRef,
     className: clsx(InputComponent.props.className, className),
-    variant,
+    ...(!input && { variant }),
```

Comment:
@ZeeshanTamboli I think it deserves a comment. Can you add a comment that explain the consition and maybe link to the issue?

Reply:
@siriwatknp I added it now.